buildSrc/build.gradle

```diff
@@ -29,10 +29,13 @@ buildscript {
         maven {
             url 'https://plugins.gradle.org/m2/'
         }
+        maven {
+            url  'http://palantir.bintray.com/releases'
```

Comment:
Ah sorry - by the way I didn't notice during the review but there's an extra space after url here. I'll fix it up but just for future reference.